CHANGELOG_PENDING.md

```diff
@@ -17,6 +17,7 @@
 ### FEATURES:
 
 ### IMPROVEMENTS:
+- [privval] \#3370 Refactored validator/kms connection (@jleni)
```

Comment:
Would be cool to list the renaming and any new introduced types here. See https://github.com/tendermint/tendermint/pull/3524/files#diff-2df96dd4ec5f206ab10b01eae20b3b21 for a good example.

Reply:
Will do. In progress.